### PR TITLE
feat(extension): add skip button during onboarding (#35)

### PR DESCRIPTION
**Pull request by randalvance**
image

### CODE REVIEW

**Comment by idoshamun** [comment]
Hi @randalvance,
First of all thank you for contributing and helping us improving Daily. It is highly appreciated.
Second, to get access to InVision please send me a message on Slack with your email address.
https://slack.dailynow.co/

**Comment by randalvance** [comment]
Updated the styles based on Invision.

File: packages/extension/src/routes/Step2.vue [hunk 1]
```diff
@@ -32,10 +32,17 @@
                  :aria-disabled="disableButton" :class="{disabled: disableButton}">
       I'm all set
     </router-link>
+    <a
```
Comment (idoshamun) [inline packages/extension/src/routes/Step2.vue]: I believe a `button` element is more appropriate for this, as this is not really a link

File: packages/extension/src/routes/Step2.vue [hunk 2]
```diff
@@ -204,4 +218,14 @@ export default {
...
   }
 }
+
+.btn-skip {
+  margin-top: 24px;
+  font-size: 11px;
+  font-weight: bold;
+  line-height: 16px;
+  letter-spacing: 1.6px;
+  color: var(--color-salt-90);
```
Comment (idoshamun) [inline packages/extension/src/routes/Step2.vue]: We have the color palette defined here: https://github.com/dailynowco/daily-apps/blob/master/packages/components/src/styles/colors.pcss
So you don't need to hardcode set the color. In this case, you should even use the theme colors and not the actual colors themselves, as it should change color according to the user's theme preference.
https://github.com/dailynowco/daily-apps/blob/master/packages/components/src/styles/themes.pcss

File: packages/extension/src/routes/Step2.vue [hunk 3]
```diff
@@ -204,4 +218,14 @@ export default {
     font-weight: 300;
   }
 }
+
+.btn-skip {
+  margin-top: 24px;
+  font-size: 11px;
```
Comment (idoshamun) [inline packages/extension/src/routes/Step2.vue]: Regarding typography styling we have them predefined.
You should choose the right typography from here, according to `font-size` and other typography properties: https://github.com/dailynowco/daily-apps/blob/master/packages/components/src/styles/mixins/typography.pcss
You can either add the class to your element or use the mixin.

**Comment by idoshamun** [comment]
Released in v2.6.0

**Pull request merged**